### ISSUE TITLE
Accessibility issues with pickadate module

### ISSUE THREAD

**Issue by wohnlice** [plone/Products.CMFPlone#2406]
We recently had an accessibility 508/WCAG2 review performed on our sites for compliance with HHS contracts and the reviewer made two notes pertaining to the pickadate jquery module used on Date/Time Dexterity fields. In my testing, this applies to both 5.0.5 and 5.1, out of the box.

I am copying his issues verbatim as I have no expertise in this area. If the Plone team believes that the current implementation is not in violation of WCAG2 for some reason, would it be possible to provide an explanation why?

**Visual Focus is Missing**
_Description_: Available days are selected after a mouse click on the date field. When tabbing and using the arrow keys, there is not a visual focus indicator to let keyboard users know which day currently has focus. This issue is a violation of 1194.21 (c) in Section 508 and Success Criterion 2.4.7 in WCAG 2.0.
_Remediation Suggestion_: Currently the year and month can be selected via keyboard. Add support so that keyboard-only users can use the arrow keys to select the day.
My note: He is referring to the date/time z3c form fields used by Dexterity and other z3c forms. This applies both to selecting the day and selecting the time.

**Comment by gp54321** [comment]
since the whole widget is meant to be axed and replaced, you may want to take a look at 
https://github.com/plone/Products.CMFPlone/issues/2150
If you could build the project in a test setup and look if it fits the norm that is interesting you, I'm sure your input on the eventual missing points could be of more interest there. That's a work in progress, now is the appropriate time to weigh in.

**Comment by jensens** [comment]
Closing as outdated (7 years old). About Pickadate accessibility in Plone 5. Plone 6 Classic UI uses pat-date-picker and Volto has its own widgets.
If you think this issue is still relevant please reopen it. Thank you for the original report!
